app/src/main/java/id/rizmaulana/covid19/ui/overview/DashboardViewModel.kt

```diff
@@ -69,5 +78,39 @@ class DashboardViewModel(
             .addTo(compositeDisposable)
     }
 
+    fun getPinUpdate() {
+        val prefData = appRepository.getPrefCountry()
+        if (prefData == null) {
+            _pinData.value = null
```

Comment:
please post the pin data through main thread delivery, with using `.postValue(...)`, or do you have any suggestions for either those ones? let's figure out.

Reply:
because of the prefData is delivered from main thread to main thread, I think there is no point making it `postValue`

Reply:
ah I see, I just realized the data comes from local storage, resolved!